installscripts/cookbooks/jenkins/recipes/configurejenkins.rb

```diff
@@ -36,14 +24,16 @@
   only_if { node['platform_family'].include?('rhel') }
   command "sudo sed -i 's/JENKINS_JAVA_OPTIONS=.*.$/JENKINS_JAVA_OPTIONS=\"-Djava.awt.headless=true -Xmx1024m -XX:MaxPermSize=512m\"/' /etc/sysconfig/jenkins"
 end
+#Docker container has no service jenkins
+if node['scenario'] == 'scenario1'
+  service 'jenkins' do
+    action :restart
+  end
 
-service 'jenkins' do
-  action :restart
-end
-
-# Wait a bit, Java apps don't coldboot very quickly...
-execute 'waitForFirstJenkinsRestart' do
-  command 'sleep 30'
+  # Wait a bit, Java apps don't coldboot very quickly...
+  execute 'waitForFirstJenkinsRestart' do
+    command 'sleep 30'
+  end
```

Comment:
I think this restart was here so that the plugins would be recognized after copying them.

Can you try deleting this restart/wait block and moving it to the very end of the `setupjenkins` cookbook instead?

Reply:
Done